client/landing/stepper/declarative-flow/internals/steps-repository/subscribers/index.tsx

```diff
@@ -51,14 +51,17 @@ const Subscribers: Step = function ( { navigation } ) {
 							flowName="onboarding_subscribers"
 							submitBtnName={ submitButtonText }
 							onImportFinished={ handleSubmit }
+							onSkipBtnClick={ handleSubmit }
 							onChangeIsImportValid={ ( isValid ) => setIsImportValid( isValid ) }
-							allowEmptyFormSubmit={ true }
+							allowEmptyFormSubmit={ false }
```

Comment:
Is this feature in the form component needed at all anymore?

Alternatively you could allow empty submits, if folks submit empty form you could just continue as if they pressed skip. No strong opinion way or the other.

Reply:
The reason I'm setting this to false is that it is needed to disable the button based on the logic of the component. 

We could leave the submit button enabled and I can skip changing this prop. 

Or we could alter the component and be sure other places that use the component/prop work as affected. 

As it was, I was already making more changes than expected and was just trying to minimize changes to only what was needed for the PR and to work with the component as written. 

But I could could go different ways on this depend on what we'd like to see. I don't have strong feelings.

Reply:
My sense on this is I'd probably not update the component itself for this PR (ie, removing that prop and adjusting other uses of the component if needed). But we could just not use the prop here, and allow users to click either the 'Add and Continue' button or the skip link. But I'd probably keep this as is - if users can click the Add button to continue, it seems to negate the need for the 'skip' link in the first place. I think that's basically the logic behind how this was originally implemented.

Reply:
Yep, fine as-is! thanks for elaborating